Doberman/Database.py

```diff
@@ -243,20 +225,43 @@ def set_pipeline_value(self, name, kvp):
     def get_message_protocols(self, level):
         """
         Gets message protocols for the specified alarm level. If none are found,
-        takes those from the next lowest level.
+        takes those from the highest level defined.
 
         :param level: which alarm level is in question (0, 1, etc)
         :returns: list of message protocols to use
         """
-        doc = self.read_from_db('alarm_config',
-                                {'level': level}, onlyone=True)
-        if doc is None:
-            self.logger.error(('No message protocols for alarm level %i! '
-                               'Defaulting to next lowest level' % level))
-            doc = self.read_from_db('alarm_config',
-                                    {'level': {'$lte': level}}, onlyone=True,
-                                    sort=[('level', -1)])
-        return doc['protocols']
+        protocols = self.get_experiment_config('alarm', 'protocols')
+        if len(protocols) < level:
+            self.logger.error(f'No message protocols for alarm level {level}! '
+                               'Defaulting to highest level defined')
+            return protocols[-1]
+        return protocols[level]
+
+    def get_message_recipients(self, level):
+        """
+        Gets message recipients for the specified alarm level
+
+        :param level: which alarm level is in question (0, 1, etc)
+        :returns: set of recipient names
+        """
+        recipient_groups = self.get_experiment_config('alarm', 'recipients')
```

Comment:
Don't forget to update the docs with info about this field

Reply:
Done